csun-metalab/nomi
# NAM-168 -- ENG: Refactor Student Images to work with the official flow

# Description
> If no likeness alpha
If no alpha official
If no official likeness default
Making the carousel a static order (likeness alpha official)
***
# Testing
* Verify carousel is displaying in correct order
* Verify image_priority is a string, not an array
* Note:  Uploading needs to be fixed in a later task
